resources/dicts/patrols/mountainous/med/newleaf.json

```diff
@@ -2990,7 +2990,7 @@
         ],
         "fail_outcomes": [
             {
-                "text": "By the time they give up and return to camp, p_l is approaching the explosion point with s_c. They've gotten in the way, questioned the use of the gathering mission, and in general been utterly obnoxious today.",
+                "text": "By the time they give up and return to camp, p_l is approaching the explosion point with s_c. They've gotten in the way, questioned the use of the gathering mission, and in general has been utterly obnoxious today.",
```

Comment:
I think in this case the original was more grammatically correct; the "has" is implied with "they've", if you want to add have to it you should add it as well to "have questions" to keep with parallel sentence structure.